bugsnag-plugin-android-ndk/src/main/jni/bugsnag.c

```diff
@@ -228,3 +228,8 @@ void bugsnag_leave_breadcrumb_env(JNIEnv *env, char *message,
   (*env)->DeleteLocalRef(env, type_class);
   (*env)->DeleteLocalRef(env, interface_class);
 }
+
+ssize_t bsg_unwind_stack_default(bugsnag_stackframe stacktrace[BUGSNAG_FRAMES_MAX],
```

Comment:
Would be nice to add some quick docs here to explain how the function is accessed for any future travellers